samples/vars/storage_fileset_vars.yml

```diff
@@ -0,0 +1,34 @@
+---
+#
+# saples/vars/storage_fileset_vars.yml
+#
+
+# Refer to `man mmchfs` and `man mmchnsd` man pages for a description of these
+# storage parameters. The `filesystem` parameter is mandatory, `servers`, and
+# the `device` parameter is mandatory for each of the file system's `disks`. All
+# other file system and disk parameters are optional.
+
+# Important:
+# `scale_storage` *must* be define using group variables. Do *not* define disk
+# parameters using host variables or inline variables in your playbook. Doing so
+# would apply them to all hosts in the group/play, thus defining the same disk
+# multiple times...
+
+scale_storage:
+  - filesystem: gpfs01
+    filesets:
```

Comment:
Can you please also mention, how user can specify fileset names. i think this should also consider multiple FSETname.